ts/Dashboards/Board.ts

```diff
@@ -583,6 +584,30 @@ class Board implements Serializable<Board, Board.JSON> {
 
         return options;
     }
+
+    /**
+     * Get a Dashboards component by its id.
+     *
+     * @returns
+     * The component with the given id.
+     */
+    public getComponentById(id: string): ComponentType | undefined {
+        return this.mountedComponents.find(
+            (c) :boolean => c.component.id === id
+        )?.component;
+    }
+
+    /**
+     * Get a Dashboards component by its cell id.
+     *
```

Comment:
Missing `@param` for the id? I assume this will go directly into the public API docs.

Reply:
@jomar-honsi it seems that @param is still missed.